sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -186,29 +186,22 @@ Metrics Advisor supports multiple types of data sources. In this sample we'll il
 string sqlServerConnectionString = "<connectionString>";
 string sqlServerQuery = "<query>";
 
-var dataFeedName = "Sample data feed";
-var dataFeedSource = new SqlServerDataFeedSource(sqlServerConnectionString, sqlServerQuery);
-var dataFeedGranularity = new DataFeedGranularity(DataFeedGranularityType.Daily);
+var dataFeed = new DataFeed();
```

Comment:
so will this break what "the guidelines" (which I can't fin written) agreed on  was about ctors and setters?
```
- Required properties on input-only models: .ctor + getter.
- Optional properties: setter + getter.
- Required properties on input-output models: .ctors + setter + getter.
```

Reply:
Yes. I think "the guidelines" you're referring to were mentioned by Krzysztof in a comment in an old PR. KC himself suggested the parameterless constructors approach in the archboard review, so I would say this is more like an exception to the rule.